src/c_wrapper.jl

```diff
@@ -212,6 +242,28 @@ function scs_solve(
     elseif m <= 0
         throw(ArgumentError("The number of constraints must be greater than 0"))
     end
+
+    # m == size(A, 1) || throw(
+    #     ArgumentError(
+    #         "The number of rows in A inconsistent with input dimension",
+    #     ),
+    # )
+    # n == size(A, 2) || throw(
+    #     ArgumentError(
+    #         "The number of cols in A inconsistent with input dimension",
+    #     ),
+    # )
+    # size(P, 1) == size(P, 2) || throw(ArgumentError("P is not square"))
+    # size(P, 1) == n || throw(
+    #     ArgumentError("P dimension $(size(P, 1)) inconsistent with n = $n"),
+    # )
```

Comment:
I don't really think we need checks like this. The MOI wrapper will validate the inputs, and if people want to call `scs_solve` directly, they can be responsible for validating.

Reply:
should we then retain checks for `m`, `n` and `bu`/`bl`? I actually intended them as more of reminders for myself when debugging scs without the library crashing ;)